pkg/querier/querier.go

```diff
@@ -106,9 +106,13 @@ func (q *V3ioQuerier) selectQry(name, functions string, step int64, windows []in
 func (q *V3ioQuerier) queryNumericPartition(
 	partition *partmgr.DBPartition, name, functions string, step int64, windows []int, filter string) (SeriesSet, error) {
 
-	mint, maxt := partition.GetPartitionRange(q.maxt)
+	mint, maxt := partition.GetPartitionRange()
 	if q.mint > mint {
 		mint = q.mint
+		if step != 0 {
+			// temporary Aggregation fix: if mint is not aligned with steps, move it to the next step tick
+			mint += (maxt - q.mint) % step
```

Comment:
you can round up with `mint += step-mint%step`
and round down with `mint -= mint%step`

but I think it is expected that reminder interval should also be counted.

Reply:
@imakhlin expected return is results with time > query mint, with your suggestion the response will not comply with the user request